application.rb

```diff
@@ -19,3 +19,7 @@
 ].each do |pattern|
   Dir.glob(Config.root.join(*pattern)).each { |file| require file }
 end
+
+::ActiveRecord::Base.schema_format = :sql
+# XXX: Update created_at and updated_at in PostgreSQL
+::Measurement.record_timestamps = false
```

Comment:
Will move it.

> Unclear on what the XXX signifies.

I am not sure I use it in right way, but

```
XXX - warn other programmers of problematic or misguiding code
```
https://en.wikipedia.org/wiki/Comment_(computer_programming)